src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+interface Props {
+  onAdd: (todo: Todo) => void;
+  todos: Todo[];
+  users: User[];
+}
+
+interface State {
+  userId: number;
+  title: string;
+  completed: boolean;
+  isUserEmpty: boolean;
+  isTitleEmpty: boolean;
+}
+
+export class NewTodo extends React.Component<Props, State> {
+  state: State = {
+    userId: 0,
+    title: '',
+    completed: false,
+    isUserEmpty: false,
+    isTitleEmpty: false,
+  };
+
+  addNewTask = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({
+      title: event.target.value.replace(/[^ a-zA-Zа-яА-я0-9]/g, ''),
+      isTitleEmpty: false,
+    });
+  };
+
+  selectUser = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    this.setState({
+      userId: +event.target.value,
+      isUserEmpty: false,
+    });
+  };
+
+  handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const { userId, title } = this.state;
+    const { users, todos, onAdd } = this.props;
+
+    if (!userId || !title.trim()) {
+      this.setState(prevState => ({
+        isUserEmpty: !prevState.userId,
+        isTitleEmpty: !prevState.title.trim(),
```

Comment:
you can write here false specifically

Reply:
Yeah, but in that case alerts will always display below both fields, even if one of them is filled